### PR TITLE
Updates to Headcount Vignette

### PR DESCRIPTION
**Pull request by JoyBaule**
This pull request is to merge the dev_branch to the master branch and is in response to [Make adjustments to Headcount Vignette Issue](https://github.com/ut-effectiveness/utDataStoR/commit/f6edce718b9c48299bd14a03fd03f80d08ff1204) https://github.com/ut-effectiveness/utDataStoR/issues/35. All changes have been made to the Headcount Vignette.

### CODE REVIEW

**Review by cdemkegit — APPROVED** [review]
These look good.

File: vignettes/headcount.Rmd
Comment (andreabr-dixie) [inline vignettes/headcount.Rmd:1]: I think these changes are great and take care of Candi's concerns.

**Review by andreabr-dixie — APPROVED** [review]

**Comment by andreabr-dixie** [comment]
@JoyBaule you should have @CSPierce01 review these changes.

**Comment by CSPierce01** [comment]
@JoyBaule the updated Headcount vignette looks great!

**Review by CSPierce01 — APPROVED** [review]
The updated Headcount vignette has been approved.

**Pull request merged**